winlogbeat/docs/fields.asciidoc

```diff
@@ -74,11 +75,11 @@ required: True
 The name of the computer that generated the record. When using Windows event forwarding, this name can differ from the `beat.hostname`.
 
 
-==== eventCategory
+==== category
 
 type: string
 
-required: True
+required: False
```

Comment:
Is that something that changed or fixing the docs? If it changed, it should be probably mentioned in the CHANGELOG.md file. Same for the similar changes below.